iceoryx_hoofs/cmake/Config.cmake.in

```diff
@@ -1,4 +1,5 @@
-# Copyright (c) 2020 by Robert Bosch GmbH All rights reserved.
+# Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
You missed the point :grin:

Reply:
The point is present on all the other bosch copyright header - so I would leave it there if you don't mind :smile:

Reply:
Oh, I might get dyschromatopsia. For some reasons I thought you removed it instead of adding it :sweat_smile: